### DISCUSSION TITLE
New Meeting not working

### DISCUSSION THREAD

**Discussion by GCITechSC** [code4recovery/12-step-meeting-list#683] [Q&A] [answered]
Hey all,  I just got an update for a new meeting for my list on aacolumbia.org.  When trying to put it in, the plugin is working fine until I put the location in.  This is an existing location that is hosting other meetings.  When I put the church name in, it brings up the address and my region (self-defined) just fine, but the map is blank, and at that point in time I lose the ability to Publish the meeting.  

I've checked on one of the groups other meetings, pulled it up as if to edit, and it is also showing a blank map.  

Any ideas where I've gone astray with this?  

Also, I just updated my computer to Windows11, but I wouldn't think that would effect this other stuff. 

Thanks in advance.
Scott

**Comment by kiyote33** [comment]
Hi Scott, 
I see from a quick look at your website that the mapbox maps are displaying correctly, so I believe you're question is more related to what might be causing the problem where you are not able to publish the new meeting.  

If you enter the location address information exactly the same as the other meetings, you should not have a problem **Publishing**.  The maps not displaying correctly on the admin page is a known issue, but does not affect the enabling/disabling of the Publish button. Normally, once the address information is validated, then the Publish button should be enabled.

If you continue to have an issue, please provide the exact details so we can try to duplicate your issue. Besides knowing the WordPress version, the 12 Step Meeting List version, and the web browser being used, the location name and address would be useful in helping find a way forward.

Thx...

**Reply by GCITechSC** [reply]
Hey all,

WP 5.9
Plugin 3.12.2
Browser is Chrome Version 98.0.4758.82 (Official Build) (64-bit)
For Geocoding the address I'm using the BETA-API Gateway. (checked this on a hunch that something else disappeared).  

Horseshoe Group is adding another meeting.  Info is as follows:
Petersen Presbyterian Church
8131 Brookfield Rd
Columbia, SC 29223

Northeast

All of that information auto populates as I fill in.  Once I enter the church name, the Publish button gets disabled.  SO weird, right?

**Reply by kiyote33** [reply]
Thanks @GCITechSC  for the update on you're issue. I was still not able to replicate the issue you are having. I can easily add the Horseshoe Group meeting in my sandbox using the info that was supplied.  In discussing this problem with the maintenance team, they suggested that the problem sounded similar to one reported earlier which they are in the process of fixing (issue #677). They expect to publish a new release within a day or two to deal with it. Please upgrade your 12 Step Meeting List plugin to the new release when it comes out and let us know if it solves your problem or not.

**Reply by GCITechSC** [reply]
Thanks all!

**Reply by GCITechSC** [reply]
Any word on a new release?

**Reply by kiyote33** [reply]
@brianw-area24 reported on a different discussion that using the BETA-API Gateway setting may be causing this problem.  They changed it to **Legacy** and it solved the problem.  Give it a try...

**Comment by jackieoh1** [comment]
I am also having this issue at nmcentraloffice.org.  No map, I can't move a draft meeting to published (it's hard to get a live "Publish" button - and when/if I do, clicking on it does nothing), and can't create a new meeting.  I look forward to the fix!

WP 5.9
Plugin 3.12.2

**Comment by GCITechSC** [comment, marked as answer]
Bang!!
